### PR TITLE
Fix for incorrect MHV URL when presented alongside Cerner URL

### PR DESCRIPTION
**Pull request by ncksllvn**
## Description
It would appear that when a user has the option of going to Cerner's system OR MHV's, we show the wrong link to MHV's system. This PR adjusts this behavior by using our MHV utility function for deriving their tool URL. 

https://github.com/department-of-veterans-affairs/va.gov-team/issues/17318

## Testing done
I logged in locally, then hardcoded some facilities onto the user profile so that every user would have a Cerner and non-Cerner facility. Then I navigated to each of the health care portals, hovered over the rendered MHV link, and confirmed that the URL was correct.

- `/health-care/get-medical-records/`
- `/health-care/refill-track-prescriptions/`
- `/health-care/secure-messaging/`
- `/health-care/view-test-and-lab-results/`

## Screenshots
Notice the URL on hover is the correct MHV environment
image

## Acceptance criteria
- [ ] Issue reported by contact at MHV is resolved

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

**Comment by ncksllvn** [comment]
I didn't realize it would call on @department-of-veterans-affairs/frontend-review-group for a review. Sorry about that. This isn't ready for full review yet.

File: src/applications/static-pages/health-care-manage-benefits/refill-track-prescriptions-page/components/AuthContent/index.js [hunk 1]
```diff
@@ -7,14 +7,22 @@ import Telephone, {
 // Relative imports.
 import CernerCallToAction from '../../../components/CernerCallToAction';
 import { getCernerURL } from 'platform/utilities/cerner';
+import { mhvUrl } from 'platform/site-wide/mhv/utilities';
 
-export const AuthContent = ({ cernerFacilities, otherFacilities }) => (
+export const AuthContent = ({
+  authenticatedWithSSOe,
```
Comment (kelsonic) [inline src/applications/static-pages/health-care-manage-benefits/refill-track-prescriptions-page/components/AuthContent/index.js:13]: Could you add this to PropTypes below? 🙂

File: src/applications/static-pages/health-care-manage-benefits/secure-messaging-page/components/App/index.js [hunk 2]
```diff
@@ -9,8 +9,13 @@ import LegacyContent from '../LegacyContent';
 import UnauthContent from '../UnauthContent';
 import featureFlagNames from 'platform/utilities/feature-toggles/featureFlagNames';
 import { selectPatientFacilities } from 'platform/user/selectors';
+import { isAuthenticatedWithSSOe } from 'platform/user/authentication/selectors';
 
-export const App = ({ facilities, showNewSecureMessagingPage }) => {
+export const App = ({
+  facilities,
+  showNewSecureMessagingPage,
+  authenticatedWithSSOe,
```
Comment (kelsonic) [inline src/applications/static-pages/health-care-manage-benefits/secure-messaging-page/components/App/index.js:17]: Could you add this to PropTypes? https://github.com/department-of-veterans-affairs/vets-website/pull/15497/files#diff-1d106f7e3518dc1d13a6de0008fa0cb1a4a4cb965da671254fd10c57dd38dd18R50

File: src/applications/static-pages/health-care-manage-benefits/view-test-and-lab-results-page/components/AuthContent/index.js [hunk 3]
```diff
@@ -7,14 +7,19 @@ import Telephone, {
...
+  cernerFacilities,
+  otherFacilities,
+}) => (
   <>
     <CernerCallToAction
       cernerFacilities={cernerFacilities}
       otherFacilities={otherFacilities}
       linksHeaderText="View lab and test results from:"
-      myHealtheVetLink="https://sqa.eauth.va.gov/mhv-portal-web/eauth"
+      myHealtheVetLink={mhvUrl(authenticatedWithSSOe, 'labs-tests')}
```
Comment (kelsonic) [inline src/applications/static-pages/health-care-manage-benefits/view-test-and-lab-results-page/components/AuthContent/index.js:22]: This helper `mhvUrl` is definitely more reliable than the hardcoded string! 😅 Thanks for doing this!

**Review by kelsonic — APPROVED** [review]
LGTM! 🎉

**Pull request merged**